src/hotspot/share/gc/shared/threadLocalAllocBuffer.cpp

```diff
@@ -105,10 +104,9 @@ void ThreadLocalAllocBuffer::accumulate_and_reset_statistics(ThreadLocalAllocSta
     stats->update_fast_allocations(_number_of_refills,
                                    _allocated_size,
                                    _gc_waste,
-                                   _fast_refill_waste,
                                    _slow_refill_waste);
   } else {
-    assert(_number_of_refills == 0 && _fast_refill_waste == 0 &&
+    assert(_number_of_refills == 0 &&
```

Comment:
Please fill back the clause from the next line. I think the assert condition now fits a single line, the text may be on the next line